test.py

```diff
@@ -0,0 +1,162 @@
+# pyGene Fusion local testing
+import json
+from pydantic import BaseModel, validator, ValidationError, StrictStr, StrictInt, StrictBool
+from typing import Optional, List, Union
+
+class Gene(BaseModel):
```

Comment:
Maybe rename to Gene Descriptor if type is GeneDescriptor? I'm wondering if a class gives a particular type if the class should be named after that type.
```suggestion
class GeneDescriptor(BaseModel):
```

Reply:
Per @ahwagner's comments on Slack, we should write out a fully-fledged Value Object Descriptor class for this, per vrsatile: https://vrsatile.readthedocs.io/en/latest/value_object_descriptor/vod_index.html#location-descriptor